configs/beaker_configs/default_finetune_multinode.yaml

```diff
@@ -62,6 +62,10 @@ tasks:
         value: true
       - name: HF_TOKEN
         secret: HF_TOKEN
+    datasets:
+      - mountPath: /oe-adapt-default
+        source:
+          weka: oe-adapt-default
```

Comment:
Is this in the single node config? Why just being added now?

Reply:
Ah good catch, just added it (also to the dpo file). It wasn't there before because we weren't commonly using files on weka, and I added it here to access the static training file. We'll also need to use our weka bucket for 70b training (downloading the 70b model as a mounted beaker dataset is way too slow), so this'll be helpful going forward as well.